lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency/expectations.js

```diff
@@ -212,13 +212,14 @@ const expectations = [
           },
         },
         'uses-responsive-images': {
-          displayValue: 'Potential savings of 53\xa0KB',
+          displayValue: 'Potential savings of 82\xa0KB',
```

Comment:
Can you remove this? It is bad for smoke tests in LR.

Reply:
heh, sure I was wondering why it was here anyway :)